src/rating-calculator/components/ChartRecordDataRow.tsx

```diff
@@ -69,6 +69,12 @@ export const ChartRecordDataRow = React.memo((props: Props) => {
                 <div key={idx}>+{r.minRt.toFixed(0)}</div>
               ))
             : '';
+        case ColumnType.NEW_RATING:
+          return record.rating
```

Comment:
Have you tried displaying Show not yet played charts? I wonder whether the code would break or not

Reply:
`Show not yet played charts` is currently broken - thanks for the catch. Will see changes required since I'm not too familiar with the score multiplication code

Reply:
Works fine now. Screenshot for both `Show played charts` and `Show not yet played charts`
<img width="1738" alt="Screenshot 2024-03-20 at 7 54 03 PM" src="https://github.com/myjian/mai-tools/assets/162117433/ec15f2df-a89f-4a73-b8f7-ee4de8f341b8">

Reply:
Let me know if there is too much code in the `case` - I will refactor to a separate function